server/controllers/alertsController.ts

```diff
@@ -27,6 +27,15 @@ export default class AlertsController {
     // Get data from middleware
     const { prisonerData, inmateDetail, alertSummaryData } = req.middleware
 
+    if (alertSummaryData.apiUnavailable) {
+      // Render banner
+      return res.render('pages/alerts/alertsPage', {
+        pageTitle: 'Alerts',
+        ...mapHeaderData(prisonerData, inmateDetail, alertSummaryData, res.locals.user, 'alerts'),
+        apiUnavailable: true,
```

Comment:
It looks like you're not using `apiUnavailable` in the template, and using `alertSummaryData.apiUnavailable` instead.  Get rid of it?